cc/modules/protocol/mpc/snn/src/impl/mpc_basic.cpp

```diff
@@ -20,6 +20,8 @@
 namespace rosetta {
 namespace snn {
 
+#define PRSS_OPT_VALUE 2
```

Comment:
Hi, could you describe the meaning of this Macro?

Reply:
This `PRSS_OPT_VALUE` means the optimization for communication with PRSS, for history reason and to make it readable, I keep the origin code and use `PRSS_OPT_VALUE`  to orgnize them.